### PR TITLE
restore license file

### PR DESCRIPTION
**Pull request by edunham**
when the new master branch was created, the LICENSE file was not brought along. This copies it from old master.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+To apply the Apache License to your work, attach the following boilerplate
+notice, with the fields enclosed by brackets "[]" replaced with your own
+identifying information. (Don't include the brackets!) The text should be
+enclosed in the appropriate comment syntax for the file format. We also
+recommend that a file or class name and description of purpose be included on
+the same "printed page" as the copyright notice for easier identification within
+third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (ElijahCaine) [inline LICENSE]: Should we change `Copyright [yyyy] [name of copyright owner]` -> `Copyright 2015 OSU Open Source Lab`?

Comment (jordane) [inline LICENSE]: Yes.

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+To apply the Apache License to your work, attach the following boilerplate
+notice, with the fields enclosed by brackets "[]" replaced with your own
+identifying information. (Don't include the brackets!) The text should be
+enclosed in the appropriate comment syntax for the file format. We also
+recommend that a file or class name and description of purpose be included on
+the same "printed page" as the copyright notice for easier identification within
+third-party archives.
+
+   Copyright 2015 OSU Open Source Lab
```
Comment (ramereth) [inline LICENSE]: This should be Oregon State University, not OSU Open Source Lab.

**Comment by lucywyman** [comment]
Changed.  lgtm if others approve. :coffee: :+1:

**Comment by lucywyman** [comment]
@ramereth Look good?

**Comment by ramereth** [comment]
:+1:

**Pull request merged**